views/userManagement.njk

```diff
@@ -15,10 +15,10 @@
     </div>
 </div>
 <br />
-<form id="searchForm" style="display:inline;" method="post">
-    <input id="searchBox" name="userInput" size="40" type="text" placeholder="Search User (by Badge ID or Email) Here"required/>
-    <input id="searchBtn" value="Go" type="submit"/>
-</form> 
+<!--form id="searchForm" style="display:inline;" method="post"-->
+    <input id="searchBox" name="userInput" size="40" type="text" onkeyup = "searchFunction()" placeholder="Search User (by Badge ID or Name) Here"required/>
+    <!--input id="searchBtn" value="Go" type="submit"/-->
+<!--/form--> 
```

Comment:
Looks good, but could you delete these commented out tags? Since they are not being used you might as well delete them.